policy-controller/src/main.rs

```diff
@@ -159,3 +177,67 @@ async fn sigterm() {
         _ => future::pending().await,
     };
 }
+
+fn log_init(LogLevel(level): LogLevel, format: LogFormat) -> Result<()> {
+    let (filter, _level) = reload::Layer::new(EnvFilter::new(level));
+    let registry = tracing_subscriber::registry().with(filter);
+
+    let dispatch = match format {
+        LogFormat::Plain => registry
+            .with(tracing_subscriber::fmt::layer().event_format(tracing_subscriber::fmt::format()))
```

Comment:
nit: we don't need to explicitly select the default format, so this can just be 
```suggestion
            .with(tracing_subscriber::fmt::layer())
```